### DISCUSSION TITLE
WMS palette conversion from V4 to V5

### DISCUSSION THREAD

**Discussion by billyz313** [Unidata/tds#346] [General]
When our team upgraded THREDDS from V4 to V5 we found that the RGB palettes needed to be converted to hex palettes. One of my team members said I wish there was a magic palette converter, so I created [https://github.com/billyz313/magic-palette-converter](https://github.com/billyz313/magic-palette-converter)  which will convert the entire directory of palettes almost instantly for anyone, eliminating the need to manually do it.  I figured I'd share it here, if you feel like it's useful, please feel free to add a link to it in your readme or wherever you like.

**Comment by lesserwhirls** [comment]
Thank you creating this and for pointing this out! A link to this is included in the `threddsConfig.xml` reference documentation section that [discusses WMS](https://docs.unidata.ucar.edu/tds/current/userguide/tds_config_ref.html#wms-service).

**Reply by billyz313** [reply]
@lesserwhirls Thank you, always glad to help where I can!
